cmd/release/README.md

```diff
@@ -0,0 +1,53 @@
+# Lotus Release Tool
+
+The Lotus Release Tool is a CLI (Command Line Interface) utility designed to facilitate interactions with the Lotus Node and Miner metadata. This tool allows users to retrieve version information in either JSON or text format. The Lotus Release Tool was developed as a part of the release process automation initiative and is used in the GitHub Actions workflows related to the release process.
+
+## Features
+
+- Retrieve version information for the Lotus Node.
+- Retrieve version information for the Lotus Miner.
+- Output logs in JSON or text format.
```

Comment:
What logs?

Reply:
Oh, I misread this.  It means the output can be printed in JSON or text.  Got it.